internal/env/env.go

```diff
@@ -1,10 +1,61 @@
 package env
 
 import (
+	"errors"
+	"fmt"
 	"os"
+	"strconv"
 	"strings"
 )
 
+const (
+	// ------------------------
+	//          auth
+	// ------------------------
```

Comment:
Maybe we could use 

```go
// #region auth
...
// #endregion auth
```

to make these foldable

Reply:
Apparently not supported for Go? https://share.cleanshot.com/5lyhDHfR

Reply:
Oh I thought I tested it and it worked but seems it only folded the start of the comment :/ Yeah doesn't seem to be supported